blocks/library/chat/style.scss

```diff
@@ -0,0 +1,20 @@
+.editor-visual-editor__block[data-type="core/chat"] {
```

Comment:
From what I've seen, other blocks use `.editor-visual-editor__block…` in `style.scss` and `wp-block-…` in `block.scss`, so I chose to follow that pattern.